addons/web/static/src/js/widgets/colorpicker.js

```diff
@@ -246,7 +313,7 @@ var Colorpicker = Dialog.extend({
                 this._updateHex(this.$('.o_hex_input').val());
                 break;
             case 'rgb':
```

Comment:
Is not this rgba now ? I see you updated the xml (maybe you should not update the xml and leave rgb here).